docs/sql/statements/start-transaction.rst

```diff
@@ -0,0 +1,42 @@
+.. highlight:: psql
+.. _ref-start:
+
+=====================
+``START TRANSACTION``
+=====================
+
+Start a transaction block
```

Comment:
```suggestion
The ``START TRANSACTION` statement starts a transaction block.
```

want to expand this a little bit. I think some of the other SQL syntax documents could do with similar improvements, but it's good to get this in early for new docs

I would also like to add a link to a page that explains what transactions are and how you can use them. I don't think we have a doc like that yet. I am guessing that is because the whole feature set is still in development

do you know if there are any existing plans to add a new doc to the [Quering](https://crate.io/docs/crate/reference/en/4.5/general/dql/index.html) section in the docs?

Reply:
Yes I agree. Aligned with your idea where you suggested to remove `The statement starts a transaction block until it committed or rolled back.` from the description, I think we should not mention `starts a transaction block` in the intro. Technically it is no-op and may confuse people. 
And also as you said, instead of explaining how to use a statement we do not support, we should mention something like "This is PostgreSQL compatibility feature that the statement will be accept but will be silently ignored."